infrastructure/template.yaml

```diff
@@ -67,6 +63,10 @@ Conditions:
     - !Not [!Equals [!Ref LambdaDeploymentPreference, AllAtOnce]]
 
 Mappings:
+  StaticVariables:
+    Urls:
+      SupportManualURL: "https://team-manual.account.gov.uk/teams/CRI-Orange-team/supporting-cri-orange/hmrc-otg-runbook/"
```

Comment:
Same question as the last PR - should we have an in-page anchor link here? As before, not hugely bothered but thought the question was worth asking

Reply:
Similar thing - we could but we'd be linking to a section that's pretty close to the top of the page
<img width="1727" alt="image" src="https://github.com/user-attachments/assets/ba8c1a5b-b5b3-4221-a7ce-e4ada9117f63" />